cluster/gce/gci/configure-helper.sh

```diff
@@ -1276,7 +1277,7 @@ clusters:
 - name: local
   cluster:
     insecure-skip-tls-verify: true
-    server: https://localhost:443
+    server: https://${KUBERNETES_MASTER_NAME:-localhost}:443
```

Comment:
KUBE_HOST_NAME? It is more consistent with the naming of other env params.

Reply:
I'm reusing the variable used by other parts of the script: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L1622

Should I create another variable for the naming sake?

Reply:
In that case no, I assumed you were creating a new param, if we already have an existing one then lets just use that.